acr_controller/service/acr_service.go

```diff
@@ -41,11 +42,12 @@ func getChangeRevisionFromRevisions(revisions []string) string {
 }
 
 func getChangeRevision(app *application.Application) string {
```

Comment:
maybe let's rename this func, because you're patching app in this method, while from `getChangeRevision` it's like getter